dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/Main.java

```diff
@@ -46,7 +55,12 @@ public static void main(String... args) throws Exception {
       main.run(args);
     } catch (MetadataDumperUsageException e) {
       LOG.error(e.getMessage());
-      for (String msg : e.getMessages()) LOG.error(msg);
+      for (String msg : e.getMessages()) {
+        LOG.error(msg);
+      }
+    } catch (Exception e) {
+      e.printStackTrace();
+      System.out.println("\nERROR\n" + extractExceptionMessages(e));
```

Comment:
Can we box this with `STARS` so that the user knows where to focus on?

Reply:
Done. The new output looks like this:

```
********************************************************************
* ERROR
* Failed to initialize pool: FATAL: password authentication failed for user "joe"
* Caused by: FATAL: password authentication failed for user "joe"
********************************************************************
```